packages/models/src/lib/implementation/schemas.ts

```diff
@@ -140,7 +142,7 @@ export function packageVersionSchema<TRequired extends boolean>(options?: {
 }
 
 /** Schema for a weight */
-export const weightSchema = nonnegativeIntSchema.describe(
+export const weightSchema = nonnegativeNumberSchema.describe(
```

Comment:
🧪 Coverage: Similarly, please replace some weights in a `group` or `category-config` schema with floats.